purchase_order_downpayment_payment_order/wizard/purchase_order_downpayment_wizard.py

```diff
@@ -0,0 +1,113 @@
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+
+
+class PurchaseOrderDownPaymentWizard(models.TransientModel):
+    _inherit = "purchase.order.down.payment.wizard"
+
+    payment_mode_id = fields.Many2one(
+        "account.payment.mode",
+        check_company=True,
+    )
+    company_id = fields.Many2one(
+        "res.company",
+        related="order_id.company_id",
+        store=True,
+    )
+
+    @api.depends("bank_partner_id")
+    def _compute_partner_bank_id(self):
```

Comment:
Where is it used ? 
I am not sure this method is needed

Reply:
I'll remove it